src/controllers/dashboard/general.js

```diff
@@ -36,6 +37,7 @@ import alert from '../../components/alert';
             config.MetadataPath = $('#txtMetadataPath', form).val();
             config.MetadataNetworkPath = $('#txtMetadataNetworkPath', form).val();
             config.QuickConnectAvailable = form.querySelector('#chkQuickConnectAvailable').checked;
+            config.ParallelImageEncodingLimit = parseInt(form.querySelector('#txtParallelImageEncodingLimit').value || '1', 10);
```

Comment:
It looks like the default in the server PR is `0`, but here you are defaulting to `1`... this should probably be `0` also.

Reply:
Fixed.